wasmcloud-wasi-kv/README.md

```diff
@@ -0,0 +1,120 @@
+# `wasmcloud-wasi-kv`
+
+This repository contains a [WebAssembly System Interface (WASI) key-value contract][wasi-kv] compliant WebAssembly module which can be composed into a "fused" component to provide key value functionality.
+
+This module is special because it implements the KV contract for [wasmCloud][wasmcloud] environments, showcasing the reusability of WASI compliant components across runtimes and platforms.
+
+## ❓ What does that even mean?
+
+- [WebAssembly ("WASM")][wasm] is a compilation target for many languages that creates a platform agnostic binary that can run on any CPU architecture, operating system, or even in the browser
```

Comment:
Silly nit, but it's "Wasm"

https://webassembly.github.io/spec/core/intro/introduction.html#wasm

Reply:
fixed!